aws.ts

```diff
@@ -117,13 +120,13 @@ mount -a
             deviceName: "/dev/sdf",
             volumeSize: 500,
             volumeType: "io2",
-            iops: 16000,
+            iops: iops,
```

Comment:
As a note, if your variable name is the same as your object key, you can just say:
```
{
    volumeType: "io2",
    iops,
}
```